### PR TITLE
Enable beta and preview environment

### PR DESCRIPTION
**Pull request by karelhala**
### Description

We want to enable hac applications on beta and preview environments (prod-beta and console.dev beta). This PR will add these caddy redirects.

### CODE REVIEW

File: Caddyfile [hunk 1]
```diff
@@ -17,7 +17,43 @@
...
+
     handle / {
         redir /apps/chrome/index.html permanent
     }
+
+    handle /preview/ {
+        redir /preview/apps/chrome/index.html permanent
+    }
+
+    handle / {
```
Comment (florkbr) [inline Caddyfile:55]: @karelhala we have this same handle twice (see just above) - is that expected? It routes to the same destination `redir /apps/chrome/index.html permanent`

Comment (karelhala) [reply Caddyfile:55]: Ah, good point! I will change that.

Comment (karelhala) [reply Caddyfile]: The last one was supposed to be `/beta` route.

**Pull request merged**